packages/dds/tree/src/test/shared-tree/opSize.bench.ts

```diff
@@ -399,7 +402,7 @@ describe("Op Size", () => {
 		resultArray: ISequencedDocumentMessage[],
 	): void {
 		// TODO: better way to hook this up. Needs to detect local ops exactly once.
-		/* eslint-disable @typescript-eslint/no-explicit-any */
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
 		const oldSubmitLocalMessage = (tree as any).submitLocalMessage.bind(tree);
```

Comment:
Nit: is there a more specific type we can cast to? Or better yet, a type-guard function we can call? Presumably `submitLocalMessage` exists on type interface definition we can leverage here instead of `any`.

Reply:
Based on our offline discussion, I see that `submitLocalMessage` is being grafted onto the object elsewhere in this file. So it's not a "real" member of any interface or anything. Closing this :)

Reply:
As per the conversation, decided to leave it as it is.